### DISCUSSION TITLE
Change point detection with STUMPY

### DISCUSSION THREAD

**Discussion by EgoVed** [TDAmeritrade/stumpy#512] [Q&A]
Good afternoon!
I have a question. I have a date frame with measurements from sensors. There are 18 sensors in total. These sensors are located on a complex unit. During operation, this unit can operate in different modes. Q: can stumpy find change point detection? Now I'm trying to use the roerich and ruptures library. I really like your library and suddenly you are faced with such a task. I can send a dataframe. Thanks a lot.

**Comment by seanlaw** [comment]
@EgoVed Thank you for your question and welcome to the STUMPY community. Typically, change point detection is interested in identifying regime changes by figuring out whether or not the statistical distribution has changed. This usually involves assessing individual data points rather than a sliding window. While STUMPY does not do classical change point detection, we do offer something called [semantic segmentation](https://stumpy.readthedocs.io/en/latest/Tutorial_Semantic_Segmentation.html) that might be useful to you in detecting that there is a local change in repeating patterns. So, the short answer is "no" but it depends on what your needs are.

I hope that helps.
